### DISCUSSION TITLE
[BUG] Error: Can't resolve '../../node_modules/@tabler/core/src/scss/fonts/inter-v7-latin-ext_latin-300.woff2'

### DISCUSSION THREAD

**Discussion by reina3596** [tabler/tabler#1044] [Q&A] [answered]
In a clean installation with Laravel I receive message:

Error: Can't resolve '../../node_modules/@tabler/core/src/scss/fonts/inter-v7-latin-ext_latin-300.woff2'

I must change de default font....

**Comment by codecalm** [comment, marked as answer]
https://github.com/tabler/tabler/pull/1048 fixes it :)
